estate/views/real_estate_views.xml

```diff
@@ -0,0 +1,14 @@
+<odoo>
+    <!-- Define the action to open your module -->
```

Comment:
No need to add those comments. They add no value. We are just describing what we are doing which we can see by looking at the code.